File: Src/java/elm/src/jvmTest/java/org/cqframework/cql/elm/serializing/ElmXmlLibraryReaderTest.java
package org.cqframework.cql.elm.serializing;

import org.hl7.elm.r1.Library;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class ElmXmlLibraryReaderTest {

    @Test
    void read() {
        var reader = new ElmXmlLibraryReader();
        Library library =
                reader.read("<library xmlns=\"urn:hl7-org:elm:r1\"></library>");
        assertNotNull(library);
    }
}

File: Src/java/elm/src/jvmTest/kotlin/org/cqframework/cql/elm/serializing/ElmXmlLibraryReaderTest.kt
package org.cqframework.cql.elm.serializing

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class ElmXmlLibraryReaderTest {
    @Test
    fun read() {
        val reader = ElmXmlLibraryReader()
        val library = reader.read("<library xmlns=\"urn:hl7-org:elm:r1\"></library>")
        Assertions.assertNotNull(library)
    }
}
